Dockerfile

```diff
@@ -36,11 +36,14 @@ RUN add-apt-repository -y ppa:cz.nic-labs/bird && \
         python-pip \
         python-pyasn1 \
         python-netaddr \
+        python-nose \
```

Comment:
Why are you installing nose into the calico-node container? You shouldn't be doing it without a really good reason...